### PR TITLE
propose contributors to SIG-archinfra

### PR DESCRIPTION
**Pull request by askhade**

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -18,6 +18,8 @@ SIG-archinfra:
     - kevinch-nv
     - winnietsang
     - chinhuang007
+    - jcwchen
+    - vinitra
```
Comment (askhade) [inline CONTRIBUTORS:22]: both Jacky and Vinitra are actively participating in resolving onnx issues, build errors and improving CI pipelines, checker, shape inference etc...

**Comment by askhade** [comment]
@onnx/steering-committee ping :)

**Comment by askhade** [comment]
@linkerzhang ping :)

**Comment by prasanthpul** [comment]
Thanks for exercising the process!

Per https://github.com/onnx/onnx/tree/master/community "A Member can become a Contributor by being sponsored by 2 existing Approvers from different companies"

Ashwini (MSFT) and Ke (Ali) are existing Arch/Infra approvers who are sponsoring Jacky and Vinitra, so we are all set here.

Welcome Jacki and Vinitra - thank you for all your previous and future contributions!

**Pull request merged**